### PR TITLE
SCH-458 # Remove master suite location from filter

### PR DESCRIPTION
**Pull request by Dyl324**

### CODE REVIEW

File: src/components/SchematixFilters.vue [hunk 1]
```diff
@@ -120,73 +120,75 @@ const orderByOptions = [
...
+            <!-- <v-col cols="12" sm="4" md="3" xl="2" class="py-0 d-none">
               <div class="d-flex flex-column">
                 <div class="pb-2">Master Suite Location</div>
                 <div class="d-flex flex-row">
                   <v-checkbox label="Front" v-model="front" @click="onClickFront"></v-checkbox>
                   <v-checkbox label="Back" v-model="back" @click="onClickBack"></v-checkbox>
                 </div>
               </div>
-            </v-col>
+            </v-col> -->
```
Comment (Zaxist) [inline src/components/SchematixFilters.vue]: Can we remove this?

File: src/components/SchematixFilters.vue [hunk 2]
```diff
@@ -14,8 +14,8 @@ const minWidth = defineModel('minWidth', { type: Number, default: undefined })
 const maxWidth = defineModel('maxWidth', { type: Number, default: undefined })
 const minLength = defineModel('minLength', { type: Number, default: undefined })
 const maxLength = defineModel('maxLength', { type: Number, default: undefined })
-const front = defineModel('front', { type: Boolean, default: undefined })
-const back = defineModel('back', { type: Boolean, default: undefined })
+// const front = defineModel('front', { type: Boolean, default: undefined })
+// const back = defineModel('back', { type: Boolean, default: undefined })
```
Comment (Zaxist) [inline src/components/SchematixFilters.vue]: Can we remove this?

**Comment by Zaxist** [comment]
Looks like you've got a few styling changes in here? Any chance of a screenshot to show the changes?

**Comment by Dyl324** [comment]
> Looks like you've got a few styling changes in here? Any chance of a screenshot to show the changes?

Screenshots on the linear task

**Pull request merged**